File: C++/Data Structures/stack/stack.cpp
#include<iostream>
#include<string>
using namespace std;
class stack{
	int size,top,arr[1000];
	public:
		stack(int size)
		{
			size = size;
			top=-1;
		}
		
		void push(int n){
			if(top == size){
				cout<<"\nStack is full";
			}else{
				arr[top+1]=n;
				top++;
			}
		}
		
		void pop(){
			if(top==-1){
				cout<<"\nStack is empty";
			}else{
				cout<<arr[top]<<"\n";
				top--;
			}
		}
		
		void display(){
			if(top==-1){
				cout<<"\nStack is empty";
			} else {
				for(int i=0;i<=top;i++){
					cout<<arr[i]<<"\n";
				}
			}
		}
};
int main()
{
	int n;
	cout<<"Enter Stack size: ";
	cin>>n;
	stack s(n);
	s.push(5);
	s.push(2);
	s.display();
	s.pop();
	s.pop();
	s.display();
	return 0;
}

File: Python/stack/stack.py

stack = []

print("1)Push\t2)Pop\t3)Display")

while True:
    n = int(input())
    if n == 1:
        print("Enter THe number: ")
        stack.append(int(input()))
    if n == 2:
        print("The Poped element: " + stack.pop())
    if n == 3:
        print(stack)
    if n > 3:
        break
